### ISSUE TITLE
SmartCity-Accessibility Server Side Network Tests #71 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#379]
Build 'SmartCity-Accessibility Server Side Network Tests' is failing!

Last 50 lines of build :
```
[...truncated 6.20 KB...]
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::        (v1.5.2.RELEASE)

-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: May 26, 2017
JxMaps license valid.
Running smartcity.accessibility.services.NavigationServiceTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 22.995 sec - in smartcity.accessibility.services.NavigationServiceTest
Running smartcity.accessibility.navigation.NavigationTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 6.771 sec - in smartcity.accessibility.navigation.NavigationTest
Running smartcity.accessibility.database.ParseDatabaseTest
Tests run: 8, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 4.531 sec <<< FAILURE! - in smartcity.accessibility.database.ParseDatabaseTest
testCount(smartcity.accessibility.database.ParseDatabaseTest)  Time elapsed: 0.278 sec  <<< FAILURE!
java.lang.AssertionError: expected:<2> but was:<5>
	at smartcity.accessibility.database.ParseDatabaseTest.testCount(ParseDatabaseTest.java:146)


Results :

Failed tests: 
  ParseDatabaseTest.testCount:146 expected:<2> but was:<5>

Tests run: 15, Failures: 1, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:33 min
[INFO] Finished at: 2017-05-26T10:04:58+00:00
[INFO] Final Memory: 32M/163M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Ariel Kolikant] 6010071be2b879a2fb8fb36777ce8125cc228792 - all the pages needed for adding locations except for the Service

- [Ariel Kolikant] f735661b6708a557bce9941f0cf8f784adc6396c - bad html last commit

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Network%20Tests/71/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Network Tests' is failing!

Last 50 lines of build :
```
[...truncated 5.15 KB...]
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::        (v1.5.2.RELEASE)

-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: May 26, 2017
JxMaps license valid.
Running smartcity.accessibility.services.NavigationServiceTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 23.488 sec - in smartcity.accessibility.services.NavigationServiceTest
Running smartcity.accessibility.navigation.NavigationTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.394 sec - in smartcity.accessibility.navigation.NavigationTest
Running smartcity.accessibility.database.ParseDatabaseTest
Tests run: 8, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 5.908 sec <<< FAILURE! - in smartcity.accessibility.database.ParseDatabaseTest
testCount(smartcity.accessibility.database.ParseDatabaseTest)  Time elapsed: 0.268 sec  <<< FAILURE!
java.lang.AssertionError: expected:<2> but was:<5>
	at smartcity.accessibility.database.ParseDatabaseTest.testCount(ParseDatabaseTest.java:146)


Results :

Failed tests: 
  ParseDatabaseTest.testCount:146 expected:<2> but was:<5>

Tests run: 15, Failures: 1, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:41 min
[INFO] Finished at: 2017-05-26T10:15:28+00:00
[INFO] Final Memory: 33M/208M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Ariel Kolikant] 6010071be2b879a2fb8fb36777ce8125cc228792 - all the pages needed for adding locations except for the Service

- [Ariel Kolikant] f735661b6708a557bce9941f0cf8f784adc6396c - bad html last commit

- [Ariel Kolikant] 44428032b400ad1e390d64e9e099681988293662 - changes as discussed with alex, not overriding toString



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Network%20Tests/72/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
